### PR TITLE
manual camera calibration documentation

### PR DESCRIPTION
**Pull request by borovik135**
a manual camera calibration documentation rst page has been drafted and referenced from index.rst

### CODE REVIEW

File: doc/manuals/cameracalibration.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+e.g. resembling the current image.
+
+To place/edit ground control points (GCP), click the 3D viewer GCP tool |gcp_button| 
+to start GCP creation and editing.
+Place six or more GCPs on the 3D model at the feature points visible from the camera view,
+observe their IDs appear in the Ground Control Points list, name them, if desired.
+
+To build 3D-2D point correspondences, click camera registration points (CRP) tool |gcp_button|
+in the Camera View.
+For each GCP in the list (also high-lighted in the 3D viewer):
```
Comment (mleotta) [inline doc/manuals/cameracalibration.rst:55]: no hyphen needed in "highlighted"

File: doc/manuals/cameracalibration.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+Choose menu: File/Import/Mesh… (Ctrl+O,M) and select an PLY file with a mesh or a point cloud,
+which should appear in the 3D viewer.
+
+Scroll to the desired video/image frame using the Camera Selection slider
+and observe it in the Camera View pane. Rotate the 3D model to the desired view,
+e.g. resembling the current image.
+
+To place/edit ground control points (GCP), click the 3D viewer GCP tool |gcp_button| 
+to start GCP creation and editing.
+Place six or more GCPs on the 3D model at the feature points visible from the camera view,
```
Comment (mleotta) [inline doc/manuals/cameracalibration.rst:50]: We need to mention that to add new points you much host the "Ctrl" key and click

File: doc/manuals/cameracalibration.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+observe their IDs appear in the Ground Control Points list, name them, if desired.
+
+To build 3D-2D point correspondences, click camera registration points (CRP) tool |gcp_button|
+in the Camera View.
+For each GCP in the list (also high-lighted in the 3D viewer):
+select a 3D point, then click on the Camera View image, where its 2D projection is expected to be.
```
Comment (mleotta) [inline doc/manuals/cameracalibration.rst:56]: also mention Ctrl + click here to add CRPs

**Review by mleotta — CHANGES_REQUESTED** [review]
This is helpful.  See a few minor inline comments.  At some point it would be nice to add screen shots to this, but we can save that for another day.

File: doc/manuals/cameracalibration.rst (hunk 1 above)
Comment (borovik135) [reply doc/manuals/cameracalibration.rst:55]: removed -

File: doc/manuals/cameracalibration.rst (hunk 2 above)
Comment (borovik135) [reply doc/manuals/cameracalibration.rst:50]: mentioned Ctrl+click

File: doc/manuals/cameracalibration.rst (hunk 3 above)
Comment (borovik135) [reply doc/manuals/cameracalibration.rst:56]: mentioned Ctrl+click

**Comment by borovik135** [comment]
addressed the review notes, added a screen shot

**Pull request merged**